### PR TITLE
Fix images loading halfway in Safari browsers

### LINKED ISSUE #8954: Issue with lazy loading of images on Russian article pages in Portrait mode on Iphone
1. Go to https://www.bbc.com/russian/news-56320777 on an iPhone.
2. There is an issue with lazy loading of images for russian article pages.
3. Some images load only partially and the only way to fix them is to turn the phone to landscape, which reloads the images, and then back to portrait. 
4. This is not hapenning on all article pages though.
IMG_2E3FC0DABAE4-1

4. Attached is the screenshot.

### PR DESCRIPTION
**Pull request by mukama**
Resolves #8954

**Overall change:**
At the moment we are using a version of [the bottom-padding hack](http://www.smashcompany.com/technology/the-bottom-padding-hack) to get around [Cumulative Layout Shift](https://web.dev/cls/) issues. How this works is, we set the image parent's height to `0` and then set the bottom-padding relative to the height of the image. The problem with that is that webkit 

This PR adds a `height` attribute to images with placeholders and sets `height: auto` to allow webkit to compute the rendered image's height better.

https://user-images.githubusercontent.com/2413911/129775616-b327b89e-e760-49a5-8a77-5fa8d5f6c32b.mp4

**Code changes:**
- Adds height attribute to images with placeholders
- Adds `height: auto` to images with placeholders

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by nicola-d-bbc — APPROVED** [review]

**Review by andrewscfc — COMMENTED** [review]
Where is this code:
> we set the image parent's height to 0 and then set the bottom-padding relative to the height of the image. The problem with that is that webkit

I can't seem to see where this is being applied?

File: src/app/containers/ImageWithPlaceholder/index.jsx [hunk 1]
```diff
@@ -9,6 +10,10 @@ import { RequestContext } from '#contexts/RequestContext';
 
 const LAZYLOAD_OFFSET = 250; // amount of pixels below the viewport to begin loading the image
 
+const StyledImage = styled(Image)`
+  height: auto;
```
Comment (jroebu14) [inline src/app/containers/ImageWithPlaceholder/index.jsx:14]: is this not the default `height`? if so then I'm not sure it needs set again

Comment (mukama) [reply src/app/containers/ImageWithPlaceholder/index.jsx:14]: It's a bit annoying but for the `CLS` hack to work we still need to explicitly specify the `height` attribute both in the `` tag and in CSS. This article attempts to explain why in detail: https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/

**Comment by mukama** [comment]
> Where is this code:
> 
> > we set the image parent's height to 0 and then set the bottom-padding relative to the height of the image. The problem with that is that webkit
> 
> I can't seem to see where this is being applied?

@andrewscfc It's applied in the ImagePlaceholder component: https://github.com/bbc/psammead/blob/f4f7a0d7fea2fe4c184b25e8864dcbc5c3aa2b2e/packages/components/psammead-image-placeholder/src/index.jsx#L18

**Review by ryanmccombe — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked few links and I am unable to recreate the issue, it's currently on LIVE. Will keep an eye to see if I come across this again, Fingers crossed ..

**Pull request merged**